plugins/plugin-sass/README.md

```diff
@@ -30,6 +30,19 @@ module.exports = {
 
 ## Plugin Options
 
-| Name     |   Type    | Description                                                                                                                                                                                                                                                                            |
-| :------- | :-------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `native` | `boolean` | If true, the plugin will ignore the npm version of sass installed locally for the native Sass CLI [installed separately](https://sass-lang.com/install). This involves extra set up, but the result can be [up to 9x faster.](https://stackoverflow.com/a/56422541) Defaults to false. |
+| Name                |   Type    | Description                                                                                                                                                                                                                                                                              |
+| :------------------ | :-------: | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `native`            | `boolean` | If `true`, the plugin will ignore the npm version of sass installed locally for the native Sass CLI [installed separately](https://sass-lang.com/install). This involves extra setup, but the result can be [up to 9× faster.](https://stackoverflow.com/a/56422541) (default: `false`). |
+| `compilerOptions.*` | `object`  | Pass [Sass options](https://sass-lang.com/documentation/cli/dart-sass#options) directly to the Sass compiler (see `compilerOptions`).                                                                                                                                                    |
+
+### `compilerOptions`
```

Comment:
lgtm! If we support all Sass compiler options, it could be good to add a link here explain that we support any not-listed here and/or how we map these the CLI args. Not sure the best way to do this without overwhelming the user, so feel free to use your best judgement here

Reply:
Some options were left-out because the plugin itself controls them (`--indent`, `--load-path`); others were left-out because they don’t make sense in the context of the plugin (`--no-color`). Realistically, though, this plugin really is passing everything to Sass, so it’s really just missing from the docs. Others, like `--input`, would probably break stuff (it’d recompile the whole world for every `.scss` file).

Should we add an explanation here? Each flag seemed so conditional, I just omitted it rather than provide a litany of different reasons.

Reply:
👍 your call

Reply:
👍 your call. To be clear,  wasn't suggesting an explanation for each. More just a top-line "below are some popular config options. We convert these to CLI arguments by doing so-and-so. See XXX for the full list"